File: bts_app/src/pinecone.js
import { Pinecone } from '@pinecone-database/pinecone';

let pineconeInstance;
let assistantId;

export const initPinecone = async () => {
  pineconeInstance = new Pinecone({
    apiKey: import.meta.env.VITE_PINECONE_KEY || "your-api-key-here"
  });
  
  // Create or retrieve assistant
  const assistants = await pineconeInstance.assistant.list();
  const existingAssistant = assistants.find(a => a.assistant_name === "bedtime-assistant");
  
  assistantId = existingAssistant?.id || (await pineconeInstance.assistant.create({
    assistant_name: "bedtime-assistant",
    instructions: "Generate children's bedtime stories with morals...",
    timeout: 30
  })).id;

  return { pineconeInstance, assistantId };
};

File: bts_app/src/pinecone.py
from pinecone import Pinecone

pc = Pinecone(api_key="pcsk_6kGHd8_TVAtD3feFZjvohdNBVwmhmarWwMMnmEvtn9sWVmHMjB3LxPn8wY1ryUPnP5wFec")

# Create Assistant
assistant = pc.assistant.create(
    assistant_name="storyteller",
    instructions="Generate children's bedtime stories",
    timeout=30
)

# Generate Response
response = pc.assistant.complete(
    assistant_id=assistant.id,
    prompt="Create a story about...",
    max_tokens=500,
    temperature=0.7
)